cmd/infra/aws/destroy.go

```diff
@@ -77,9 +80,14 @@ func NewDestroyCommand() *cobra.Command {
 	return cmd
 }
 
-func (o *DestroyInfraOptions) Run(ctx context.Context) error {
+func (o *DestroyInfraOptions) Run(ctx context.Context, timeout time.Duration) error {
```

Comment:
why do you need to pass the timeout as an argument here?
you can just get it from the DestroyInfraOptions struct as `o.AwsInfraGracePeriod`

Reply:
I don't, good spot.